packages/lingua-franca/src/templates/pages/css/community.scss

```diff
@@ -85,6 +85,10 @@
         background-color: lighten(black, 30%);
       }
       
+      &.slack {
+        background-image: url("../../../assets/community/connect-slack-icon.svg");
+        background-color: lighten(rgb(167, 167, 167), 30%);
```

Comment:
Would this work as expected in the Dark theme as well?

Reply:
To me, this looks just as good in the Dark theme. Picture below
![slack-dark](https://user-images.githubusercontent.com/7968955/174225269-edb744c8-e0af-42a7-9cb5-41d8d10c607d.png)